api/src/main/resources/org/openmrs/api/db/hibernate/ConceptReferenceRange.hbm.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+	"-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+	"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+	<class name="ConceptReferenceRange" table="concept_reference_range" batch-size="25">
+
+		<cache usage="read-write"/>
+
+		<id name="conceptReferenceRangeId" type="java.lang.Integer" column="concept_reference_range_id" unsaved-value="undefined">
```

Comment:
Is there any reason why you used a hibernate xml mapping file for this instead of annotations?

Reply:
I was getting errors on tests. But the application works well if I skip tests.

Reply:
I mean, why use both annotations and xml mapping files for the same entity?

Reply:
For me I would advocate for the annotations but I didn't get time to investigate on what could be causing failures in tests. Could you point me to some configurations I can check and resolve this?

Reply:
Just do the right thing and i will help you resolve the errors. In other words, do either the annotations or xml mappings, but not both.

Reply:
The only place I'm aware of we've needed to update this in the past is [here](https://github.com/openmrs/openmrs-core/blob/4510cb29a7c7d0ddf402b016196d9855a78f7c37/api/src/test/java/org/openmrs/api/OrderServiceTest.java#L2719) (which does some bizarre stuff with Hibernate).

Reply:
Thanks @ibacher, I will checkout for this 😄